portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,44 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <nav>
+      <label class="logo">KvngSnow's Portfolio</label>
+      <ul>
+          <li><a href="http://www.github.com/JalenJenkins">Github</a></li>
+          <li><a href="https://www.instagram.com/kvngsno">Instagram</a></li>
+          <li><a href="https://www.linkedin.com/in/jalen-jenkins-b75772186">Linkedin</a><li> 
+          </ul>
+
+    <div id="image"> <img src = "images/self image.jpg" alt = "A picture of myself" style="width:128px;height:128px;"></div>
+    
+    <div id="content"> 
+
+      <h1>Jalen's Portfolio</h1>
+      
+      <p>Welcome to my Portfolio!</p>
+      
+      <h2>About me</h2>
+      <p> 
+          <br> My name is Jalen Jenkins and I am currently a Computer Science Major in Cornell University's Class
+          of 2023. Although I have only been coding since August of 2019, I feel as though I have learned a
+          ton since starting. I am working here from home in Charlotte, North Carolina, and I am incredibly
+          excited to take in everything Google has to offer! <br><br>Born and raised in Charlotte, North Carolina, 
+          I've always been a curious person. When I was younger, I had a passion for science, so my parents
+          pushed me in the premed direction. It was not until January of 2019 when I came to the realization that
+          pursuing medicine would not satisfy me in life! I then realized that I valued my time and freedom,
+          which prompted me to pursue a career that would allow me to have both: Entrepreneurship! I now look for
+          opportunities to have multiple streams of income. <br><br> </p>
+      
+      <p>Click the button below to get a random hidden talent I possess:</p>
+    
```

Comment:
In general, I'd suggest removing a lot of the extra blank lines in here. Not all, but I try to use them when it helps make a clean logical grouping and when it helps visually break up a large chunk of text. And when using them, stick to one. That will make the final file feel tighter for people looking at it later.